wpm/game.py

```diff
@@ -290,6 +291,8 @@ def handle_key(self, key):
             # Finished typing?
             if self.position == len(self.quote.text):
                 self.mark_finished()
+        elif self.hard_flag:
+            self.reset()
```

Comment:
Does this work properly with backspace and ESC? I just see that the incorrect counter is updated below in the other IF block. Perhaps that's where it should be? (I haven't _tried_ your PR, so you may be right, I'm just asking if everything else works)

Reply:
Just tested both with hard flag and without the hard flag once more to see ESC and backspace behavior.

both versions have:
- ESC resets the level if playing, exits the program if not playing.
- Backspace allows deleting the current word but won't go past (or before) the current word.
- Meta-backspace resets the level.

during the hard mode the second elif statement is never reached.
- the if block accepts the correct keystrokes
- elif hard.flag accepts everything else

during the regular mode
- the if block accepts the correct keystrokes
- elif hard.flag is always false so the next elif block is run.

I would like to think everything else works with it.